content/jobstos.md

```diff
@@ -0,0 +1,69 @@
+---
+title: PyBaMM Job Board Terms and Conditions
+shortcutDepth: 1
+summary: "Terms and conditions for posting jobs on the PyBaMM Jobs Board"
+---
+
+## Introduction
+
+These Terms and Conditions govern the job posting services offered by PyBaMM, a project under [NumFOCUS, Inc.](https://numfocus.org). Companies agree to pay a fee for job postings related to battery modeling positions on PyBaMM’s job board. All income generated from this service is considered earned income and is mission-aligned.
+
+---
+
+## Payment for Job Postings
+
+A fee of **$250** will be charged for each job posting. Academic positions (e.g., postdoctoral research assistants) will be charged a discounted rate of **$125**, and PhD positions a further discounted rate of **$62.5**.
+
+Payments will be processed via the [PyBaMM website](https://www.pybamm.org). Upon payment, the job will be posted for a duration of up to **90 days** (or until the closing date, whichever is earlier).
+
+Optionally, companies may make an additional **suggested donation** to support PyBaMM’s ongoing work.
+
```

Comment:
"Payments will be processed via the PyBaMM website" might be slightly inaccurate; we don't link to a payment aggregator directly, we just have an embed for payment providers in the "Donate" page back when I added it last year (while we don't use GitHub Pages right now, but such things are against the GitHub ToS). Are the payments going to be done through OpenCollective (https://pybamm.org/donate/#opencollective), @brosaplanella? 

We could point payers to the relevant page directly and ask them to add a "jobs board" note when they proceed with their payment (I was not involved with this, of course, so I'm not aware of the specifics)

Reply:
I'd wait for @brosaplanella on content decisions as suggested :)
Feel free to push changes on my branch.

Reply:
NumFOCUS will setup a different payment link (Open Collective is for donations and this is technically a commercial transaction. In any case, once we have the payment link we will have to update this.